### DISCUSSION TITLE
It does not appear MinVer sets InformationalVersion, why?

### DISCUSSION THREAD

**Discussion by nietras** [adamralph/minver#920] [General]
Is there a reason why MinVer does not set InformationalVersion?

**Comment by adamralph** [comment]
@nietras [MinVer sets the `Version` SDK property](https://github.com/adamralph/minver#version-numbers), which is the [default value used for `InformationalVersion`](https://andrewlock.net/version-vs-versionsuffix-vs-packageversion-what-do-they-all-mean/#informationalversion).

**Comment by nietras** [comment]
@adamralph right, probably thrown off by usual usage and that 
```xml
  <Target Name="AfterMinVer" AfterTargets="MinVer">
    <Message Text="AssemblyVersion: $(AssemblyVersion)" Importance="high"/>
    <Message Text="FileVersion: $(FileVersion)" Importance="high"/>
    <Message Text="InformationalVersion: $(InformationalVersion)" Importance="high"/>
    <Message Text="PackageVersion: $(PackageVersion)" Importance="high"/>
    <Message Text="MinVerVersion: $(MinVerVersion)" Importance="high"/>
    <Message Text="MinVerMajor: $(MinVerMajor)" Importance="high"/>
    <Message Text="MinVerMinor: $(MinVerMinor)" Importance="high"/>
    <Message Text="MinVerPatch: $(MinVerPatch)" Importance="high"/>
    <Message Text="MinVerPreRelease: $(MinVerPreRelease)" Importance="high"/>
    <Message Text="MinVerBuildMetadata: $(MinVerBuildMetadata)" Importance="high"/>
  </Target>
```
will show `InformationalVersion` as nothing. If you change this to `BeforeTargets="Build"` then it will be set. So one just have to be careful when to use this then.

**Reply by adamralph** [reply]
Note that if you switch [`MinVerVerbosity` to `detailed`](https://github.com/adamralph/minver#can-i-get-log-output-to-see-how-minver-calculates-the-version) you will see the values of all the variables set by MinVer in the build output. But no, [`InformationalVersion` is not one of them](https://github.com/adamralph/minver/blob/bc89aa1c9ff94653960cfc9fbd1ed950fe9a3a6d/MinVer/build/MinVer.targets#L68-L77). That property is set by the SDK in a later target.
